library/std/src/sys/theseus/os.rs

```diff
@@ -43,6 +64,7 @@ where
     I: Iterator<Item = T>,
     T: AsRef<OsStr>,
 {
+    // Theseus doesn't have the concept of a `PATH` environment variable.
```

Comment:
can we log/print/throw a more detailed error here just to catch when this is used?

Reply:
`std` doesn't ever log or print messages, but it is hypothetically possible for Theseus because we link to the log crate using `-L` flags.

Reply:
fair. Perhaps we do need to add a `PATH` concept, but we can always save that for later, e.g., when we support standard executables.